src/app/legacy/components/Promo/timestamp.jsx

```diff
@@ -21,7 +21,7 @@ const PromoTimestamp = ({ children, serviceDatetimeLocale, className }) => {
       padding={false}
       service={service}
       timezone={timezone}
-      isRelative={isTenHoursAgo(children)}
+      isRelative={isTenHoursAgo(new Date(children).getTime())}
```

Comment:
Is it worth extracting `isRelative` to a function so that it matches the `TimestampContainer`?